tests/integration/test_igv.py

```diff
@@ -0,0 +1,167 @@
+import requests
+import dash
+import dash_bio
+import dash_html_components as html
+
+from common_features import simple_app_layout
+
+_COMPONENT_ID = 'myigv'
+
+igvStyle = dict(
+    paddingTop='10px',
+    paddingBottom='10px',
+    margin='8px',
+    border='1px solid lightgray'
+)
+
+
+def test_dbigv001_ASM985889v3(dash_duo):
+    app = dash.Dash(__name__)
+
+    try:
+        requests.get('https://www.google.com/').status_code
+        data_path = 'https://s3.amazonaws.com/igv.org.genomes/'
+    except requests.exceptions.ConnectionError:
+        print("Running test with local datasets")
+        data_path = app.get_asset_url('')
```

Comment:
Is there a reason not to always use the local datasets in the test?

Also, nonblocking but looks like it would be a little simpler to include the subdiretory and base filename in `data_path` - so below you'd just have `data_path + ".fna"` for example.

Reply:
I think it's important to use an online dataset for at least one of the tests since it seems the more common use-case for  this component would be to load a remotely hosted database. It might be worthwhile to make sure that functionality doesn't break, especially since the `genome` prop pulls data from a list of pre-defined reference genomes hosted by igv. 

On a similar note, based on a [comment by the developer of igv.js here](https://bioinformatics.stackexchange.com/a/10666), and the way local file loading is setup in their web app, it does seem like Dash simplifies loading local files significantly.

Reply:
OK. We've had issues in the past where some external resource went down and broke our CI tests, but I suppose `s3` going down would take a large fraction of the internet with it, so probably the least of our worries 😅